posthog/special_migrations/test/test_runner.py

```diff
@@ -0,0 +1,126 @@
+from datetime import datetime
+
+import pytest
+from django.db import connection
+
+from posthog.models.special_migration import MigrationStatus, SpecialMigration
+from posthog.models.utils import UUIDT
+from posthog.special_migrations.examples.test import Migration
+from posthog.special_migrations.runner import (
+    attempt_migration_rollback,
+    run_special_migration_next_op,
+    start_special_migration,
+)
+from posthog.special_migrations.test.util import create_special_migration
+from posthog.test.base import BaseTest
+
+TEST_MIGRATION_DESCRIPTION = Migration().description
+
+
+class TestRunner(BaseTest):
+    def setUp(self):
+        create_special_migration(name="test", description=TEST_MIGRATION_DESCRIPTION)
+        return super().setUp()
+
+    # Run the full migration through
+    @pytest.mark.ee
+    def test_run_migration_in_full(self):
```

Comment:
could we check that the sql op was also completed, just 1 quick check whatever is easiest to distinguish from when we didn't run ops at all (the `self.assertEqual(res, ("a", "b"))`).